core/src/main/scala/com/databricks/labs/remorph/parsers/FunctionBuilder.scala

```diff
@@ -236,4 +238,15 @@ object FunctionBuilder {
         ir.UnresolvedFunction(name, args, is_distinct = false, is_user_defined_function = false)
     }
   }
+
+  /**
+   * Functions can be called even if they are quoted or bracketed. This function removes the quotes and brackets.
+   * @param str
+   *   the possibly quoted function name
+   * @return
+   *   function name for use in lookup/matching
+   */
+  private def removeQuotesAndBrackets(str: String): String = {
+    str.replaceAll("^['\"\\[]|['\"\\]]$", "")
```

Comment:
Unfortunately, this is still incorrect (for instance `"[x]` becomes `[x`).
```suggestion
    val quotations = Map('\'' -> "'", '"' -> "\"", '[' -> "]", '\\' -> "\\")
    str match {
      case s if s.length < 2 => s
      case s =>
        quotations.get(s.head).fold(s){ closingQuote =>
          if (s.endsWith(closingQuote)) {
            s.substring(1, s.length - 1)
          } else {
            s
          }
        }
    }
```